src/lib/protocols/proof/messages/ProofRequestMessage.ts

```diff
@@ -0,0 +1,41 @@
+export class ProofRequestMessage {
```

Comment:
I'm a little bit confused about what this class should represent? Should it be this object https://github.com/hyperledger/aries-rfcs/blob/master/features/0037-present-proof/README.md#presentation-preview?

Reply:
No, although we can used for that as well. ProofRequestMessage.ts is used inside RequestPresentationMessage to send proof request. 
**Please find below link, -> section Step 3.2: Acme sends Proof Request to Alice and Bob**
https://kctheservant.medium.com/demonstration-of-hyperledger-aries-cloud-agent-6e476a5426b0
-------------------------------------------------
sample message is provided as below,
{
  "connection_id": "ba1a486f-14ce-4835-a0b6-d42c6c7a2c3b",
  "proof_request": {
    "name": "Proof of Education",
    "version": "1.0",
    "requested_attributes": {
      "0_name_uuid": {
        "name": "name",
        "restrictions": [
          {
            "cred_def_id": "VA3GvaAZRZwrH4dDtVWd6E:3:CL:8:default"
          }
        ]
}
    },
    "requested_predicates": {
      "0_average_GE_uuid": {
        "name": "average",
        "p_type": ">=",
        "p_value": 4,
        "restrictions": [
          {
            "cred_def_id": "VA3GvaAZRZwrH4dDtVWd6E:3:CL:8:default"
          }
        ]
      }
    }
  }
----------------------------------------
 , I used above as reference and that information is store in ProofRequestMessage  and RequestPresentationMessage is wrapper for that to store ProofRequestMessage as base64 format, comment & id